kolibri/plugins/facility_management/assets/src/views/user-page/reset-user-password-modal.vue

```diff
@@ -1,68 +1,50 @@
 <template>
 
-  <core-modal
+  <k-modal
     :title="$tr('resetPassword')"
+    :submitText="$tr('save')"
+    :cancelText="$tr('cancel')"
+    :submitDisabled="isBusy"
+    @submit="submitForm"
     @cancel="displayModal(false)"
   >
-    <form @submit.prevent="submitForm">
+    <p>{{ $tr('username') }}: <strong>{{ username }}</strong></p>
```

Comment:
the `:` should be moved inside the `username` string

(I realize this was a pre-existing issue)